src/Control/Effect/Carrier.hs

```diff
@@ -19,9 +19,9 @@ class HFunctor h where
   {-# INLINE fmap' #-}
 
   -- | Higher-order functor map of a natural transformation over higher-order positions within the effect.
+  -- A definition for 'hmap' over first-order effects can be derived with the @-XDeriveAnyClass@ extension.
```

Comment:
You don’t actually have to use `-XDeriveAnyClass`; you can also do `instance HFunctor MyEffect` w/o a `where` clause. Might be worth noting that?

Reply:
Good call!